frontends/web/src/api/exchanges.ts

```diff
@@ -24,6 +24,38 @@ export type AddressSignResponse = {
   address: string;
   error: string;
 }
+export type ExchangeRegionList = {
```

Comment:
this does not cover the error case which the backend can respond with

Reply:
There is no error anymore. See my previous comment